webapp/src/main/frontend/components/organisms/ControlPanel.test.tsx

```diff
@@ -1,15 +1,17 @@
-import {fireEvent, render, screen, waitFor, within} from "@testing-library/react";
+import {fireEvent, screen, waitFor, within} from "@testing-library/react";
 import {ControlPanel} from "./ControlPanel";
 import {Builder} from "builder-pattern";
 import {Management, Thing, ThingStatus, ThingType} from "../../logic/Types";
 import '@testing-library/jest-dom';
+import {render} from "../../testUtils/TestUtils";
+import {TextDecoder, TextEncoder} from 'util';
```

Comment:
MAJOR: Is there a way to avoid this repeated in all the tests? Why jest config is not working locally?